.github/workflows/integration_tests.yml

```diff
@@ -115,6 +116,15 @@ jobs:
           # Build and tag a docker container
           docker build --tag aleph-node:latest -f ./docker/Dockerfile .
 
+      - name: Generate chainspec
+        run: |
+          # NOTE : remove this step after keys are moved to a separate JSON file
+          cp docker/data/n_members /tmp/n_members
+          cp docker/data/authorities_keys /tmp/authorities_keys
```

Comment:
`authorities_keys` are generated by `dev-keys` below, so this command may be removed.

Reply:
well, this leads to a panic, becasue of a kind of catch22 situation:
```
   0: rust_begin_unwind
             at /rustc/a178d0322ce20e33eac124758e837cbd80a6f633/library/std/src/panicking.rs:515:5
   1: core::panicking::panic_fmt
             at /rustc/a178d0322ce20e33eac124758e837cbd80a6f633/library/core/src/panicking.rs:92:14
   2: core::panicking::panic_bounds_check
             at /rustc/a178d0322ce20e33eac124758e837cbd80a6f633/library/core/src/panicking.rs:69:5
   3: aleph_node::chain_spec::testnet1_config
   4: aleph_node::command::<impl sc_cli::SubstrateCli for aleph_node::cli::Cli>::load_spec
   5: aleph_node::cli::CLiDevKeys::run
   6: aleph_node::command::run
   7: aleph_node::main
```

it's because CLiDevKeys -> open_keystore -> load_spec -> testnet1_config -> read_keys :/

Reply:
Sudo account...Jedrzej is fixing this right now so you may add another NOTE above

Reply:
too late.. refactored :)

Reply:
Nice :muscle: